resources/views/index.blade.php

```diff
@@ -6,11 +6,17 @@
 
         window.laravelCookieConsent = (function () {
 
+            var COOKIE_VALUE = 1;
+
             function consentWithCookies() {
-                setCookie('{{ $cookieConsentConfig['cookie_name'] }}', 1, 365 * 20);
+                setCookie('{{ $cookieConsentConfig['cookie_name'] }}', COOKIE_VALUE, 365 * 20);
                 hideCookieDialog();
             }
 
+            function cookieExists(name) {
+                return (document.cookie.split(';').indexOf(name + '=' + COOKIE_VALUE) > 0);
```

Comment:
First approach (`== 0`) worked when I tested it in my project because it was actually the first cookie

Second approach (`> 0`) I just tested in the browser to confirm it successfully retrieved a cookie that wasn't first 😑 

Tested this change in my own project and it resolve that issue you pointed out

**But** I've noticed another problem. If I go to a page belonging to my project and press back, it works. If I go to an external website and hit back, it doesn't.

It looks like there's a space being prefixed to the name in that scenario. I'm not sure if this is a JS quirk or something in the script. I'll try to figure out what's going on and send another update.